src/vuegraf/vuegraf.py

```diff
@@ -138,6 +138,29 @@ def extractDataPoints(device, usageDataPoints):
                     usageDataPoints.append(createDataPoint(account, chanName, watts, timestamp, True))
                     index += 1
 
+def extractHistoricalDataPoints(device, startTime, stopTime, usageDataPoints):
```

Comment:
There's a lot of duplicated logic with the above `extractDataPoints` function. Did you consider adjust the other function to accept start/stop params, and perhaps a historic flag? The primary concern with the duplicated code is that a future PR might not realize there are two functions that perform similar duties and so one could get updated, but not the other.

Reply:
You're definitely right that it would be better for all of the logic to be contained within a single function. I chose to use `extractDataPoints ` for historical data whenever the `startTime` and `endTime` flags are specified, instead of a `history` flag. I prefer this because it is less redundant than creating a new variable only for the purpose of denoting `history` "mode", when the other two variables are only set for this purpose.